tensorboard/compat/proto/update.sh

```diff
@@ -26,11 +26,20 @@ rsync --existing "$1"/tensorflow/core/protobuf/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/core/profiler/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/core/util/*.proto tensorboard/compat/proto/
 rsync --existing "$1"/tensorflow/python/framework/*.proto tensorboard/compat/proto/
-if [ -d $1"/tensorflow/tsl/protobuf" ] # This directory first appears in TF 2.11
+# This directory first appears in TF 2.11 and is removed at commit:
+# https://github.com/tensorflow/tensorflow/commit/132c98b29c1a42298cd13055d1168c8d67177bc9
+if [ -d $1"/tensorflow/tsl/protobuf" ]
```

Comment:
Can you explain for which scenarios we would want to maintain backward compatibility? The only reason I could think of is for patch releases on old branches. But in that case could they use the version of the script that exists in that release branch?

Reply:
I think you're right... I did it because that's what we did the previous time, I guess, when we added that if statement... but I agree, this can be updated to the current state of things, and the script can remain simpler.